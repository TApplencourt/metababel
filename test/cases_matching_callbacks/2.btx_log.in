event_1: { callback = "" }
