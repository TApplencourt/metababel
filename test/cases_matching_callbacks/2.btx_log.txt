[23:20:02.668833240] (+?.?????????) event_1: { cf_1 = "test" }, { pf_1 = 0 }
[23:20:02.668837077] (+0.000003837) event_1: { cf_1 = "test" }, { pf_1 = 1 }
[23:20:02.669013249] (+0.000176172) event_1: { cf_1 = "test" }, { pf_1 = 2 }
[23:20:02.669019918] (+0.000006669) event_1: { cf_1 = "test" }, { pf_1 = 3 }
