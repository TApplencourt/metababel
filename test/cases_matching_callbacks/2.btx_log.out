event_1: { callback = "usr_foo_callback" }
event_1: { callback = "usr_bar_callback" }
